.eslintrc.json

```diff
@@ -25,7 +25,28 @@
         "/coverage/**"
     ],
     "rules": {
-        "simple-import-sort/imports": "error",
+        "simple-import-sort/imports": [
+            "error",
+            {
+                "groups": [
```

Comment:
Do we need something similar for Organize Imports collation?

Reply:
Maybe, but it's not strictly required to be "allowed" to use this, because as of #48330 we don't move imports between groups.